.buildkite/pipeline_ab.py

```diff
@@ -15,11 +15,17 @@
 )
 
 perf_test = {
-    "block": {
-        "label": "🖴 Block Performance",
+    "virtio-block": {
+        "label": "🖴 Virtio Block Performance",
         "test_path": "integration_tests/performance/test_block_ab.py",
         "devtool_opts": "-c 1-10 -m 0",
     },
+    "vhost-user-block": {
+        "label": "🖴 vhost-user Block Performance",
+        "test_path": "integration_tests/performance/test_block_vhost_user_ab.py",
+        "devtool_opts": "-c 1-10 -m 0",
+        "timeout_in_minutes": 40,
```

Comment:
Why the timeout?

Reply:
I think these are rebase artifacts, good catch. We removed the timeouts from the pipeline steps a while ago, since we rely on the per-test pytest timeouts here.

Reply:
fixed as a part of Patrick's comment.